resources/views/widgets/availability-map.blade.php

```diff
@@ -0,0 +1,24 @@
+@extends('layouts.widget')
+
+@section('content')
+
+<center><span data-toggle="tooltip" title="" class="badge bg-green" data-original-title="{{ $count['up'] }} Devices up">{{ $count['up'] }}</span> <span data-toggle="tooltip" title="" class="badge bg-yellow" data-original-title="{{ $count['warn'] }} Recently rebooted">{{ $count['warn'] }}</span> <span data-toggle="tooltip" title="" class="badge bg-red" data-original-title="{{ $count['down'] }} Devices down">{{ $count['down'] }}</span></h5></center>
+
+<?php $count = ['warn' => 0, 'up' => 0, 'down' => 0]; ?>
+@foreach ($devices as $device)
+
+    @if ($device->status == 1)
+        @if ($device->uptime < $uptime) && ($device->uptime != '0'))
```

Comment:
We use $uptime in the controller as well, thought it was better to just call settings once and pass the variable where we needed.